src/components/SwitchField.tsx

```diff
@@ -53,6 +56,14 @@ export const SwitchField: FC<FieldProps<SwitchFieldSchema>> = ({
     return null;
   }
 
+  useEffect(() => {
+    if (value) {
+      onDisable && onDisable(setValue, reset, values);
```

Comment:
This syntax is a bit weird because you're using a regular if statement then switching to `&&`

I think a clearer way to write it would be:

```
if(!value){
return
}

if(onDisable){
onDisable(setValue, reset, values);
}

if(onEnable){
onEnable(setValue, reset, values);
}
```